docset/windows/storage/Get-PhysicalDisk.md

```diff
@@ -129,7 +129,7 @@ Accept wildcard characters: False
 
 ### -CimSession
 Runs the cmdlet in a remote session or on a remote computer.
-Enter a computer name or a session object, such as the output of a [New-CimSession](http://go.microsoft.com/fwlink/p/?LinkId=227967) or [Get-CimSession](http://go.microsoft.com/fwlink/p/?LinkId=227966) cmdlet.
+Enter a computer name or a session object, such as the output of a [New-CimSession](https://docs.microsoft.com/powershell/module/cimcmdlets/new-cimsession?view=powershell-5.1) or [Get-CimSession](https://go.microsoft.com/fwlink/p/?LinkId=227966) cmdlet.
```

Comment:
Remove the `?view-powershell-5.1` as part of the link to make it generic to the current version.

Reply:
Ok, from my experience without the version it leads to PowerShell 6 but maybe when you are already viewing 2016 it will go to 5.1.